packages/core/src/dialects/abstract/query-generator.d.ts

```diff
@@ -93,13 +94,13 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
     valueHash: object,
     columnDefinitions?: { [columnName: string]: NormalizedAttributeOptions },
     options?: InsertOptions,
-  ): { query: string; bind?: unknown[] };
+  ): { query: string; bind?: Record<string, unknown> };
```

Comment:
could you move `{ query, bind }` to its own reusable type?

Reply:
Done.